Dev/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js

```diff
@@ -151,7 +160,9 @@ export class ClassMainScreen extends QcParentScreen {
               <TopBanner
                 LeftIconName="navicon"
                 LeftOnPress={() => this.setState({ isOpen: true })}
+                isEditingTitle={this.state.isEditing}
                 Title={this.state.currentClass.name}
+                onTitleChanged={(newTitle)=> this.updateTitle(newTitle)}

```

Comment:
In this if clause, there should be a rightIconName and a rightOnPress because the class name should be editable even if the class doesn't have any students